feature-configs/base/ipv6/02_network_attachment_defintion.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: openshift.io/sriovnic
+  name: dual-stack-net-attachment-def
+  namespace: dual-stack-testing
```

Comment:
Should this match the namespace we use in the sriov network (like this one https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/demo/sriov/dpdk-network.yaml#L5 ) ?

Also, I wonder if it would make sense to create these in the test itself.

Reply:
Done